app/src/main/java/com/hedvig/app/feature/embark/passages/textactionset/TextActionFragment.kt

```diff
@@ -95,57 +97,52 @@ class TextActionFragment : Fragment(R.layout.fragment_text_action_set) {
         delay(PASSAGE_ANIMATION_DELAY_MILLIS)
     }
 
-    private fun createInputViews(): List<View> {
-        return data.keys.mapIndexed { index, key ->
-            val inputView = LayoutInflater.from(requireContext()).inflate(R.layout.embark_input_item, binding.inputContainer, false)
-            val inputLayout = inputView.findViewById<TextInputLayout>(R.id.textField)
-            val inputEditText = inputView.findViewById<TextInputEditText>(R.id.input)
+    private fun createInputViews(): List<View> = data.keys.mapIndexed { index, key ->
+        val inputView = EmbarkInputItemBinding.inflate(layoutInflater)
```

Comment:
We can still pass `container` and `attachToParent` here for more efficient layouting 🙂